README.md

```diff
@@ -31,6 +31,7 @@ There should be a `yarn` script for each package found in `packages`.
 ```bash
 $ yarn test:x-domain
 $ yarn test:tx-ingestion
+$ yarn test:sequencer-interactions
```

Comment:
Since this list is starting to get longer, maybe we should put the content of these commands into a `test.js` script and allow a list of package names to be specified so we could do something like `yarn test x-domain tx-ingestion`.